<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1872</width>
    <height>981</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>1731</width>
      <height>931</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 12pt &quot;Lucida Sans&quot;;

background-color: rgb(112, 113, 112);
color: rgb(0, 0, 0);

</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1727</width>
       <height>889</height>
      </rect>
     </property>
     <attribute name="label">
      <string>Milestone 3</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>611</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Lucida Sans&quot;;
background-color: rgb(197, 197, 197);</string>
      </property>
      <property name="title">
       <string>Select Location</string>
      </property>
      <widget class="QComboBox" name="stateList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>40</y>
         <width>64</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>City</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>64</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>State</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>40</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Zipcode</string>
       </property>
      </widget>
      <widget class="QListWidget" name="cityList">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>70</y>
         <width>221</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QListWidget" name="zipList">
       <property name="geometry">
        <rect>
         <x>440</x>
         <y>70</y>
         <width>151</width>
         <height>151</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>0</y>
        <width>501</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Lucida Sans&quot;;
background-color: rgb(197, 197, 197);</string>
      </property>
      <property name="title">
       <string>Zipcode Statistics</string>
      </property>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>50</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string># of Businesses:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Total Population:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>170</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Average Income:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="businessesLine">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>50</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="populationLine">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>110</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="incomeLine">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>170</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="totalBSlabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:url(&quot;C://Users//namej//Desktop//NamJunLee_milestone3//picture//stat.jpg&quot;);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="totalPPlabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:url(&quot;C://Users//namej//Desktop//NamJunLee_milestone3//picture//population.jpg&quot;);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="avgLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>41</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:url(&quot;C://Users//namej//Desktop//NamJunLee_milestone3//picture//income.png&quot;);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>1210</x>
        <y>0</y>
        <width>491</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Lucida Sans&quot;;
background-color: rgb(197, 197, 197);</string>
      </property>
      <property name="title">
       <string>Top categories</string>
      </property>
      <widget class="QTableWidget" name="categoriesTable">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>451</width>
         <height>201</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>311</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Lucida Sans&quot;;
background-color: rgb(197, 197, 197);</string>
      </property>
      <property name="title">
       <string>Filter on Categories</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 10pt &quot;Lucida Sans&quot;;</string>
       </property>
       <property name="text">
        <string>Select Category</string>
       </property>
      </widget>
      <widget class="QListWidget" name="categoriesList">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>281</width>
         <height>192</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>250</y>
        <width>1351</width>
        <height>251</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Lucida Sans&quot;;
background-color: rgb(197, 197, 197);
</string>
      </property>
      <property name="title">
       <string>              Businesses</string>
      </property>
      <widget class="QTableWidget" name="businessTable">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>1231</width>
         <height>201</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
      <widget class="QPushButton" name="searchButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:url(&quot;C://Users//namej//Desktop//NamJunLee_milestone3//picture//search.jpg&quot;);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="clearButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>51</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image:url(&quot;C://Users//namej//Desktop//NamJunLee_milestone3//picture//clean.png&quot;);
background-color: rgb(255, 255, 255);
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>520</y>
        <width>851</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Lucida Sans&quot;;
background-color: rgb(214, 214, 236);</string>
      </property>
      <property name="title">
       <string>Popular Businesses (zipcode)</string>
      </property>
      <widget class="QTableWidget" name="popularTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>831</width>
         <height>291</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(246, 247, 255);</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>960</x>
        <y>520</y>
        <width>741</width>
        <height>341</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 11pt &quot;Lucida Sans&quot;;
background-color: rgb(212, 180, 169);</string>
      </property>
      <property name="title">
       <string>Successful Businesses (zipcode)</string>
      </property>
      <widget class="QTableWidget" name="successfulTable">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>721</width>
         <height>291</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 251, 247);</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="refreshButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>520</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image:url(&quot;C://Users//namej//Desktop//NamJunLee_milestone3//picture//refresh.png&quot;);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1872</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
